Add mapping service ui

ajaykirarkit opened the pull request.

Pfeil commented:
Currently it does not work die to "@latest" still pointing to the old version, but it should update soon. For trying it out, you may replace it with "@0.5.3" locally until jsdeliver updated this. I used the purge cache function already and assume that we are in some kind of queue to get this fixed.

Pfeil commented:
Seems to work for me now. @ajaykirarkit can you try it out?

Pfeil reviewed (commented):
I only took a quick look, as I ran out of time. But looks good in my browser so far. I did not tet functionality, but I assume you did so. Logo etc. is still missing, of course. Please look at my comments.

ajaykirarkit commented:
> 
for logo we have a selected image but we need .svg file , i tried creating on adobe it didnt work well .. can you please help me in creating svg file .. below is link to free image 

https://www.freedigitalphotos.net/images/Other_business_conce_g200-Paper_For_Ideas_p41213.html

Pfeil commented:
- The logo at the top left is still the one of the typed pid maker in mapping-service-ui.html.
- The default base-url is to the nffa mapping service. Consider making it http://localhost:<default port here> or something.

Pfeil commented:
> I want to use previous mapping service in order to recieve map file ( which is reflected in search bar ) so if we change url to localhost or somthing we wont recieve the map file ..

I would say that is why we added an address bar, but lets see what the results will be on thursday.

> I changed the image bt it doesnt reflect the mapping service logo on the header .

> You can change it in mapping-service-settings.js. Doesn't it work?

ajaykirarkit commented:
> mapping-service-settings.js

My bad :( .. it worked when i did changes in mapping-service-settings.js

ajaykirarkit commented:
> I would say that is why we added an address bar, but lets see what the results will be on thursday.
> Aah got it .. i changed base url to http://localhost:8095/
